test/integration/testsuite/testsuite.go

```diff
@@ -778,3 +784,11 @@ func directoryShouldntExist(dir string) error {
 
 	return fmt.Errorf("Directory %s exists", dir)
 }
+
+func setEnvironmentVariable(key string, value string) error {
+	return os.Setenv(key, value)
+}
+
+func unSetEnvironmentVariable(key string) error {
```

Comment:
`os.Unsetenv` returns an error when it fails.. is that handled by the `s.Step`?

Reply:
@anjannath Yes, s.Step requires and handles the error, when error is not nil, the step fails. Error is reported, scenario ended as failed and the feature continues with next scenario.

Reply:
hmm...Let me check

Reply:
@amitkrout It seems `s.Step` already takes care of it as @agajdosi mentioned in the above-1 comment.

Reply:
I did not know that, @agajdosi thanks for the info

Reply:
i misread the comment https://github.com/minishift/minishift/pull/2360#discussion_r188190110 . yes s.step handles the error